docker_run.sh

```diff
@@ -53,20 +53,48 @@ else
     cd src/
     ln -s ../build CMakeFiles
     cd ..
+    
+    BUILD_START=$(date)
+    echo $BUILD_START
+    
 # run cmake separately so config.h will be seen by the md5 script
     cmake -DCMAKE_BUILD_TYPE=Release -DENABLE_GAME=true -DENABLE_SERVER=true -DENABLE_CAMPAIGN_SERVER=true -DENABLE_TESTS=true -DENABLE_NLS=false -DEXTRA_FLAGS_CONFIG="-pipe" -DEXTRA_FLAGS_RELEASE="$EXTRA_FLAGS_RELEASE" -DENABLE_STRICT_COMPILATION="$STRICT"
 # run manual md5 file tracking/mtime modification script for cmake
     python3 cmake_mtime_crc.py
+    PY_RES=$?
+    if [ "$PY_RES" != "0" ]; then
+      rm build/cmake_mtime_crc.txt
+      echo "Failed to update cmake file mtimes!"
+      exit 1
+    fi
     
-    make VERBOSE=1 -j2
+# filter out spammy "is newer than dependee"/"is newer than depends" output that can occasionally fail the build because of its sheer volume
+    make VERBOSE=1 -j2 | grep -v "is newer than"
```

Comment:
I'm worried about just filtering out those messages. I think they're not supposed to occur in the first place.

Reply:
The messages themselves aren't hard to make appear and I think are normal, for example you can see a bunch by doing:
`cmake -DCMAKE_BUILD_TYPE=Release -DENABLE_NLS=no && touch src/config.h && make -j 6 VERBOSE=1`

What definitely shouldn't happen is having more than 4 MBs worth of those messages appearing, but finding any documentation at all about these messages or the `depend.internal` files and the like has been difficult.  It's also inconsistent on travis, and I haven't been able to reproduce it locally, so that doesn't help either.